.github/workflows/test.yml

```diff
@@ -1,21 +1,20 @@
-name: lint_python
+name: test
 on: [pull_request, push]
 jobs:
-  lint_python:
+  test:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
           python-version: 3.x
-      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
```

Comment:
If we are going to go this way then we should make a full https://pre-commit.com so that tests are run the same locally and in CI

Reply:
why is this related? correct me if I am wrong, but prior to this PR there was no pre-commit hook

Reply:
There was none but if we are going to make `pip install tap2junit[test]` then we should provide a way to use all those testing dependencies locally.  If this is not done then there is noo reason to move the configuration out of the file that is running the tests.

Reply:
I see, I really have no strong opinion regarding this - do you prefer adding a pre-commit hook or moving these back into this file?

Reply:
Let's do this...  We can land this as is and then I will circle back and create a pre-commit when we add support for Python 3.11 (due next month).

Reply:
👍🏻